### ISSUE TITLE
Angular generate-proxy does not work on api that return file

### ISSUE THREAD

**Issue by ericeric284** [abpframework/abp#13111]
Api generated by cli command "abp generate-proxy -t ng" will fail, if the api returns file.
A possible soluction is by setting api response type as blob.
I hope there can be some support for this type of api.

## Version info

* ABP Framework version: 5.3.0 
* User Interface: Angular
* Database provider: EF Core

## Step to reproduce

1. Create template with angular by abp cli (bookstore for example)
2. Create a controller with sample api that returns file content like example below
```` C#
 [HttpGet]
  [Route("sample-file")]
  public IActionResult SampleFile()
  {
      var txtBytes = System.Text.Encoding.ASCII.GetBytes("Hello world");

      return File(txtBytes, "text/plain", $"sample.txt");
  }

  [HttpGet]
  [Route("sample-file-stream")]
  public IRemoteStreamContent SampleFileStream()
  {
      var txtBytes = System.Text.Encoding.ASCII.GetBytes("Hello world");

      var stream= new MemoryStream(txtBytes);

      return new RemoteStreamContent(stream, $"sample.txt", "text/plain");
  }
````
3. Create Angular proxy with the following command
````
abp generate-proxy -t ng
````
generated service:
````JS
import { RestService } from '@abp/ng.core';
import { Injectable } from '@angular/core';
import type { IActionResult } from '../microsoft/asp-net-core/mvc/models';
import type { IRemoteStreamContent } from '../volo/abp/content/models';

@Injectable({
  providedIn: 'root',
})
export class SampleFileService {
  apiName = 'Default';

  sampleFile = () =>
    this.restService.request<any, IActionResult>({
      method: 'GET',
      url: '/sample-file',
    },
    { apiName: this.apiName });

  sampleFileStream = () =>
    this.restService.request<any, IRemoteStreamContent>({
      method: 'GET',
      url: '/sample-file-stream',
    },
    { apiName: this.apiName });

  constructor(private restService: RestService) {}
}

````

4. Exceute the generated proxy's functions in angular,expect it to return blob,but it will throw excpetion.
![image](https://user-images.githubusercontent.com/74721170/175559457-9c3b802f-89d1-4e14-bfc1-e51590c73b24.png)

````
"SyntaxError: Unexpected token H in JSON at position 0
    at JSON.parse (<anonymous>)
    at XMLHttpRequest.onLoad (http://localhost:4200/vendor.js:50117:39)
    at _ZoneDelegate.invokeTask (http://localhost:4200/polyfills.js:3526:31)
    at Object.onInvokeTask (http://localhost:4200/vendor.js:76727:33)
    at _ZoneDelegate.invokeTask (http://localhost:4200/polyfills.js:3525:60)
    at Zone.runTask (http://localhost:4200/polyfills.js:3298:47)
    at ZoneTask.invokeTask [as invoke] (http://localhost:4200/polyfills.js:3607:34)
    at invokeTask (http://localhost:4200/polyfills.js:4781:18)
    at globalCallback (http://localhost:4200/polyfills.js:4824:33)
    at XMLHttpRequest.globalZoneAwareCallback (http://localhost:4200/polyfills.js:4845:16)"
````

A workaround is to add responseType in api
```` JS
  SampleFileStream = () =>
    this.restService.request<any, IRemoteStreamContent>({
      method: 'GET',
      url: '/sample-file-stream',
      responseType: Rest.ResponseType.Blob //add this line
    },
    { apiName: this.apiName });
````

**Comment by mahmut-gundogdu** [comment]
The issue is fixed on rel-6.0. See the PR. https://github.com/abpframework/abp/pull/13561

**Comment by snowchenlei** [comment]
@ericeric284 angular how download?
